### PR TITLE
Ogone - v 2.13

### PR DESCRIPTION
**Pull request by studiokiwik**
- module configuration page modified
- translations added / modified
- version up to 2.13
- minor cleanups & bugfixes
- ORIG variable added

### CODE REVIEW

**Comment by gaillafr** [comment]
Hi @studiokiwik,
Trying to install the new version, I have an "empty" layout:
https://www.dropbox.com/s/c3vlq8i77t3wvk7/Screenshot%202015-06-22%2017.24.56.png?dl=0

Kind regards.

**Comment by studiokiwik** [comment]
Tabs are all clickable, but good point, I will fix it ASAP

**Comment by studiokiwik** [comment]
It's ok, tabs are visible by default

File: upgrade/upgrade-2.13.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	exit;
+
+/**
+ * Function used to update your module from previous versions to the version 1.1,
+ * Don't forget to create one file per version.
+ */
+function upgrade_module_2_13($module)
+{
+	return $module ? true : false;
+}
```
Comment (Quetzacoalt91) [inline upgrade/upgrade-2.13.php]: This file can be removed because it does nothing.

**Comment by Quetzacoalt91** [comment]
Hi,

You made so many templates for prices.tpl and info.tpl ! You could only have one of them by implementing the translations inside of it.

File: upgrade/upgrade-2.13.php (hunk 1 above)
Comment (studiokiwik) [inline upgrade/upgrade-2.13.php]: Ok, it's removed

**Comment by studiokiwik** [comment]
concernig templates - there is a lot of text in those templates, and it is a lot easier to manage translated text in separate files

**Comment by Quetzacoalt91** [comment]
Hi @studiokiwik,

Okay for the templates, I really hope that you won't forget to update each of them. :)
I just merged your PR and the changes will be available from tomorrow.

Best regards

**Pull request merged**